app/auth/sign-out.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
+import React, { useState } from "react";
+import { Button, TextField, Grid, Typography, Paper, Box } from "@mui/material";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { emailSchema } from "app/utils/schemas";
+import { sendPasswordlessLoginEmail, signInWithLink } from "app/api/auth";
+import { useRecoilState } from "recoil";
+import { UserState } from "app/recoil-store";
+
+export const SignIn: React.FC<{}> = () => {
+	// useForm & useAuth initialization
+	const {
+		register,
+		handleSubmit,
+		formState: { errors }
+	} = useForm<{ email: string }>({
+		resolver: yupResolver(emailSchema)
+	});
+
+	// state used to detect if email sent
+	const [sentEmail, setSentEmail] = useState(false);
+
+	// get user state
+	const [user, setUser] = useRecoilState(UserState);
+
+	const submitEmail = async (data: { email: string }) => {
```

Comment:
add like 1 or 2 lines of documentation to this. When I skim code I mostly look for comments //